extension-qr-code/.project

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>extension-qr-code</name>
```

Comment:
Why don't we have folder name and project name in sync with `Bundle Symbolic Name?

Reply:
Thanks for looking at the code. I'll try to change the code accordingly. BTW most of what you say is because I took the RotatedText plugin as a template.